docs/UserGuide.md

```diff
@@ -225,35 +273,51 @@ Format: `addmem n/MEMBER_NAME hp/PHONE e/MEMBER_EMAIL a/MEMBER_ADDRESS [ag/ALLER
+</div>
+
+<div style="border: 1px solid #FFA500; margin: 10px 0; padding: 10px; background-color: #FFE8B2; color: #CC6C00; border-radius: 5px;">
+  ⚠️ <strong>Warning:</strong> Ensure `MEMBER_NAME` is unique, preferably the full name, if not this might cause trouble in the future when using other commands like `addorder`.
+</div>
 
 ### 4.4 Listing all members : `list`
 
 Shows a list of all members in the loyalty program.
 
 Format: `list`
 
-🌟 <span style="color:Gold">**Benefit**:</span> Stay on top of your customer management by using list to view all your loyalty program members at a glance. This can help you quickly access customer information and make informed decisions on rewards and promotions.
+<div style="border: 1px solid #D1478C; margin: 10px 0; padding: 10px; background-color: #FDE4EF; color: #910D47; border-radius: 5px;">
+  🧁 <strong>Benefit:</strong> Stay on top of your customer management by using list to view all your loyalty program members at a glance. This can help you quickly access customer information and make informed decisions on rewards and promotions.
+</div>
 
 ### 4.5 Adding an Item to the Catalogue : `additem`
 
 Adds an item along with its corresponding points to the catalogue.
 
 Format: `additem i/ITEM_NAME p/POINTS_PER_UNIT`
 
-* `ITEM_NAME` must be a unique identifier for the item.
-* `POINTS_PER_UNIT` represent the number of points a member earns upon purchasing the item.
+* `ITEM_NAME` must be a unique identifier for the item. It should only contain alphabets and spaces.
+* `POINTS_PER_UNIT` represent the number of points a member earns upon purchasing the item. It must be a positive integer 1, 2, 3, …
 
 Examples:
 * `additem i/Chocolate Cake p/50` Adds `Chocolate Cake` with `50` points to the catalogue.
 
-🌟 <span style="color:Gold">**Benefit**:</span> Enhance your loyalty program's rewards by adding new items to the catalogue. This allows you to keep your offerings fresh and enticing, encouraging more purchases and participation from members.
+<div style="border: 1px solid #D1478C; margin: 10px 0; padding: 10px; background-color: #FDE4EF; color: #910D47; border-radius: 5px;">
+  🧁 <strong>Benefit:</strong> Enhance your loyalty program's rewards by adding new items to the catalogue. This allows you to keep your offerings fresh and enticing, encouraging more purchases and participation from members.
+</div>
+
+<div style="border: 1px solid #FFA500; margin: 10px 0; padding: 10px; background-color: #FFE8B2; color: #CC6C00; border-radius: 5px;">
+  ⚠️ <strong>Warning:</strong> You are strongly recommended to only use points less than 1000 to ensure the total points/membership points a member has remains within the safe range of 1…2,000,000,000.
```

Comment:
hmm maybe can remove the `1000`, or change to a larger number `10000`? Still need 2 million orders of 1000 points to hit the 2 billion